### PR TITLE
Add my planck layout

### PR DESCRIPTION
**Pull request by abhixec**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/abhixec/config.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#pragma once
```
Comment (drashna) [inline keyboards/planck/keymaps/abhixec/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/planck/keymaps/abhixec/readme.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# The Default Planck Layout
```
Comment (drashna) [inline keyboards/planck/keymaps/abhixec/readme.md:1]: Probably should remove or update this

**Review by drashna — APPROVED** [review]

File: keyboards/planck/keymaps/abhixec/config.h [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+   - MIDI notes can be added to the keymap
+   - Octave shift and transpose
+   - Virtual sustain, portamento, and modulation wheel
+   - etc.
+*/
+//#define MIDI_ADVANCED
+
+/* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
+//#define MIDI_TONE_KEYCODE_OCTAVES 2
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/abhixec/config.h:48]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/planck/keymaps/abhixec/keymap.c [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+        tap_code(KC_MS_WH_UP);
+      #else
+        tap_code(KC_PGUP);
+      #endif
+    }
+  }
+    return true;
+}
+
+void dip_switch_update_user(uint8_t index, bool active) {
```
Comment (drashna) [inline keyboards/planck/keymaps/abhixec/keymap.c:200]:
```suggestion
bool dip_switch_update_user(uint8_t index, bool active) {
```

File: keyboards/planck/keymaps/abhixec/keymap.c [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+            break;
+        }
+        case 1:
+            if (active) {
+                muse_mode = true;
+            } else {
+                muse_mode = false;
+            }
+    }
+}
```
Comment (drashna) [inline keyboards/planck/keymaps/abhixec/keymap.c:229]:
```suggestion
    return true;
}
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**